### ISSUE TITLE
resolves ammonite for ages when run from sbt 1.0-RC2

### ISSUE THREAD

**Issue by antonkulaga** [coursier/coursier#626]
It resolves ammonite 1.01 for Scala 2.12.3 for ages when run from sbt 1.0-RC2 (speed 0 bytes per second) while everything seems to work fine with the previous version of sbt and with other dependencies

**Comment by antonkulaga** [comment]
I also have issue resolving scala-js-java-time
```
libraryDependencies += "org.scala-js" %%% "scalajs-java-time" % "0.2.2"libraryDependencies += "org.scala-js" %%% "scalajs-java-time" % "0.2.2"
```
is resolved find with coursies under sbt 0.13.x but when switching to sbt 1.0-RC2 it cannot be found

**Comment by alexarchambault** [comment]
About ammonite, it worked fine on a quick test for me. Do you have more details (small repro, ...)?

About scalajs-java-time, https://github.com/coursier/coursier/pull/627/commits/4e74e7f66dbe45e165ca696829eecb8b8b020d72 fixes it (will be included in `1.0.0-RC9`).

**Comment by alexarchambault** [comment]
Closing this. Re-open if you have more details.
